client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -146,6 +161,12 @@ function CheckoutSummaryFeaturesList( props ) {
 		refundDays = 4;
 	} else if ( hasPlanInCart && ! hasDomainsInCart ) {
 		refundDays = hasMonthlyPlan ? 7 : 14;
+	} else if ( isPaidSite ) {
```

Comment:
This logic should be safe because if `isPaidSite` is true, then `sitePlan` should also exist, but the code above doesn't necessarily guarantee that. Fortunately a simple guard here for `sitePlan` should make sure we don't accidentally try to use its properties if it doesn't exist.

```suggestion
	} else if ( isPaidSite && sitePlan ) {
```

I wonder, actually, if we even need `isPaidSite` here? What about just:

```suggestion
	} else if ( sitePlan ) {
```

Either way, it's fine.